client/service/models/Users.js

```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose'
+
+// Creating a schema
+const UserSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: [true, 'First name is required']
+    },
+    lastName: {
+        type: String, 
+        required: [true, 'Last name is required']
+    },
+    userName: {
+        type: String,
+        required: [true, 'Username is required']
+    },
+    password: {
+        type: String,
+        required: [true, 'Password is required']
+    },
+    emailAddress: {
+        type: String,
+        required: [true, 'Email address is required']
+    },
+    decks: [{
```

Comment:
This is a good User model. We'll explore something different in class this week, but this is a perfectly viable option.